src/components/views/read/cvForm/Review.tsx

```diff
@@ -26,27 +27,9 @@ const Review: FC<Props> = ({
   prevStep,
   cvValues,
 }) => {
-  const personalData = cvValues.personalInfo;
-  const educationData = cvValues.educationInfo;
-  const workData = cvValues.workInfo;
-  const skillsData = cvValues.skillsInfo;
-  const portfolioData = cvValues.portfolioInfo;
-  const motivationData = cvValues.motivationInfo;
-  const referencesData = cvValues.referencesInfo;
-  const templateData = cvValues.templateInfo;
-
+  const professionalTemplate = <FirstTemplate cvValues={cvValues} />;
```

Comment:
Why do you define this here ? I thought that in the review you show the CV with the chose template ?